### ISSUE TITLE
Error when loading project in bimsurfer3. WebGL not support.

### ISSUE THREAD

**Issue by dcapeletti** [opensourceBIM/BIMsurfer#80]
Hello, when I go to http://localhost:8080/apps/bimsurfer3/apps/dev.html and click on the project, the following message appears:
**Unable to initialize WebGL. Your browser or machine may not support it.**

However if I go to https://get.webgl.org/ it tells me that my browser has support for webgl.

Does this happen to you? How can I solve my problem?
Attached is a picture of the error.

Using Firefox 68.10.0esr (64-bit) on Debian 9.
![BimSurfer error webgl](https://user-images.githubusercontent.com/5890886/101941122-5032c500-3bc6-11eb-9198-8083deb1f873.png)

**Comment by aothms** [comment]
This version uses webgl2, what's the info on this page https://get.webgl.org/webgl2/ ?

**Comment by dcapeletti** [comment]
About wegl2 it says:
`Hmm. While your browser seems to support WebGL2, it is disabled or unavailable. If possible, please ensure that you are running the latest drivers for your video card.`

However when I go to https://get.webgl.org/webgl2/enable.html to determine the webgl2 configuration neither chrome nor firefox work. I think it's a problem with my video card. My video card is Intel HD Graphics 3000.
```

VGA compatible controller [0300]: Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller [8086:0116] (rev 09)
	Subsystem: Acer Incorporated [ALI] 2nd Generation Core Processor Family Integrated Graphics Controller [1025:0506]
	Kernel driver in use: i915
	Kernel modules: i915
```

glxinfo | grep version

```
server glx version string: 1.4
client glx version string: 1.4
GLX version: 1.4
    Max core profile version: 3.3
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.0
OpenGL core profile version string: 3.3 (Core Profile) Mesa 13.0.6
OpenGL core profile shading language version string: 3.30
OpenGL version string: 3.0 Mesa 13.0.6
OpenGL shading language version string: 1.30
OpenGL ES profile version string: OpenGL ES 3.0 Mesa 13.0.6
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.00
```


I really have no idea if my card supports webgl2. If you have any help I will be grateful.

Attached is a picture of the configuration I made in chrome.
![webgl2](https://user-images.githubusercontent.com/5890886/102011221-a5cfb480-3d21-11eb-8fb6-3e4dd5e6635d.png)

**Comment by aothms** [comment]
Yes, for some time we can expect these kind of issues (especially on Linux). I would pick this up with the community of Firefox/Chrome and see if you they can guide you to resolve this. This isn't specific to BIMsurfer.
